src/excmds.ts

```diff
@@ -3281,30 +3281,38 @@ export async function yankimage(url: string): Promise<void> {
 
 /** Change active tab.
 
-    @param index
-        Starts at 1. 0 refers to last tab of the current window, -1 to penultimate tab, etc.
+    @param id
+        A string following the following format: "[0-9]+.[0-9]+", the first number being the index of the window that should be selected and the second one being the index of the tab within that window.
 
-        "#" means the tab that was last accessed in this window
+        A bare number means the current window is used.
```

Comment:
I think 99% of people will still be using it in this way, so this should come first. The x.y format is almost an aside by comparison